replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  summarise(fivehundred.go = (sum(go) / n()) * 100)
+
+clean_2b %>%
+  filter(split == 0, cond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()) * 100)
+
+```
+
+```{r echo = FALSE}
+clean_2b_high_300 <- clean_2b %>%
+  filter(split == 1, cond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2b_high_500 <- clean_2b %>%
+  filter(split == 1, cond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2b_high_1000 <- clean_2b %>%
+  filter(split == 1, cond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (2b) - high income
+```
+
+```{r echo = FALSE}
+clean_2b_low_300 <- clean_2b %>%
+  filter(split == 0, cond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2b_low_500 <- clean_2b %>%
+  filter(split == 0, cond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2b_low_1000 <- clean_2b %>%
+  filter(split == 0, cond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (2b) - low income
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(clean_2b_low_300$threehundred.go, clean_2b_high_300$threehundred.go, clean_2b_low_500$fivehundred.go, clean_2b_high_500$fivehundred.go, clean_2b_low_1000$onethousand.go, clean_2b_high_1000$onethousand.go)
+tablet_cost = c("$300", "$300", "$500", "$500", "$1,000", "$1,000")
+CIs = c(clean_2b_low_300$CI, clean_2b_high_300$CI, clean_2b_low_500$CI, clean_2b_high_500$CI, clean_2b_low_1000$CI, clean_2b_high_1000$CI)
+
+fig2b.table <- data.frame(title, values, tablet_cost, CIs)
+#figure 2b data frame
+fig2b.table$tablet_cost <- factor(fig2b.table$tablet_cost, levels = c("$300", "$500", "$1,000"))
+
+figure2b <- fig2b.table %>%
+  ggplot(aes(x = title, y = values, fill = tablet_cost)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("proportion willing to travel") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + ggtitle("Figure 2b - Proportion of participants willing to travel\nfor a discounted tablet computer in Study 2b\n(error bars represent 95% confidence interval)") + theme_bw() + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 2b
+
+print(figure2b)
+#displays figure 2b
+```
+
+
+> The ineraction between income and context was significant in a binary logistic regression, beta = -0.48, Walt-test chi square (1, N = 702) = 4.93, p < 0.05,
```

Comment:
rather than highlight significance, try interpreting in plain words what someone would expect from the data.